packages/TesterApp/react-native.config.js

```diff
@@ -1,3 +1,20 @@
+const project = (() => {
+  try {
+    const { configureProjects } = require('react-native-test-app');
+    return configureProjects({
+      android: {
+        sourceDir: 'android',
+      },
+      ios: {
+        sourceDir: 'ios',
+      },
+    });
+  } catch (_) {
+    return undefined;
+  }
+})();
+
 module.exports = {
+  ...(project ? { project } : undefined),
   commands: require('@callstack/repack/commands'),
 };
```

Comment:
Since you're not publishing TesterApp, you can simplify the config:
```suggestion
const { configureProjects } = require('react-native-test-app');

module.exports = {
  project: configureProjects({
    android: {
      sourceDir: 'android',
    },
    ios: {
      sourceDir: 'ios',
    },
  }),
  commands: require('@callstack/repack/commands'),
};
```